Cargo.toml

```
@@ -5,6 +5,8 @@ edition = "2021"
 
 [features]
 bench = []
+test-utils = []
+redis-store = ["redis"]
```

I prefer the term memory which refers to the implemented trait rather than store that does not need much.
```suggestion
redis-mem = ["redis"]
```